src/components/mobile-menu-container/index.tsx

```diff
@@ -17,7 +17,7 @@ import Button from 'components/button'
 import ButtonLink from 'components/button-link'
 
 // Local imports
-import { ThemeSelectWithLabel } from 'components/theme-switcher'
+import ThemeSelectWithLabel from 'components/theme-switcher'
```

Comment:
📌 Can we align the default exported component and file name as follow-up when this is stable?

Reply:
Sure, I keep waffling on the name 😆  Do you have thoughts on what this should be called?

Reply:
I think `theme-switcher` + `ThemeSwitcher` makes sense! We've used similar names for context-switch-y type components before, and if they're all in Swingset makes them slightly easier to search by "switcher"

Reply:
Sweet, makes sense!